core/src/main/kotlin/fr/sncf/osrd/api/api_v2/path_properties/PathPropRequest.kt

```diff
@@ -0,0 +1,26 @@
+package fr.sncf.osrd.api.api_v2.path_properties
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import fr.sncf.osrd.api.api_v2.pathfinding.TrackRange
+import fr.sncf.osrd.utils.json.DistanceAdapterFactory
+import fr.sncf.osrd.utils.json.OffsetAdapterFactory
+
+class PathPropRequest(
+    @Json(name = "track_section_ranges") val trackSectionRanges: List<TrackRange>,
+    @Json(name = "rolling_stock_supported_electrifications")
+    val rollingStockSupportedElectrifications: List<String>,
```

Comment:
```suggestion
```

I would rather get the client to perform this filtering pass. it's not a lot of data, and it enables figuring out why a rolling stock can't go somewhere, in addition to whether it can

Reply:
We can already figure out why a rolling stock can't go somewhere, using the pathfinding endpoint.

Reply:
We discussed the matter and came to the conclusion that it's best to remove this argument, as:
- it allows for easier caching, using only (infra, path)
- it makes the API less error prone, as the client can't mistake unsupported electrified zones for non-electrified zones

We also considered changing the result API to make the second point clearer, but decided against it because of the first point